src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -137,7 +137,17 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
                 continue;
             }
 
-            if ((false === $propertyMetadata->isReadableLink() || false === $propertyMetadata->isReadable()) && false === $propertyMetadata->getAttribute('fetchEager', false)) {
+            if ($inAttributes = isset($context[AbstractNormalizer::ATTRIBUTES][$association])) {
+                // prepare the child context
+                $context[AbstractNormalizer::ATTRIBUTES] = $context[AbstractNormalizer::ATTRIBUTES][$association];
+            } else {
+                unset($context[AbstractNormalizer::ATTRIBUTES]);
+            }
+
+            if (
+                ((!$inAttributes && false === $propertyMetadata->isReadableLink()) || false === $propertyMetadata->isReadable()) &&
+                false === $propertyMetadata->getAttribute('fetchEager', false)
```

Comment:
` false === $propertyMetadata->getAttribute('fetchEager', false)` should not be needed here as it should've been tested https://github.com/dunglas/core/blob/master/src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php#L142 

Did you start you PR on master? I can't recognize the replaced code which may have lead to weird copy/paste?

Reply:
Can I let you handle that? I've changed this part (only the CS) and I'm not really comfortable with it.